tests/functional/conftest.py

```diff
@@ -0,0 +1,79 @@
+import errno
+import os
+import shutil
+import socket
+import subprocess
+import tempfile
+
+import pytest
+from model_config import CPU, GPU, MODEL_CONFIG, models
+
+CONTAINER_NAME = "openvino_backend_pytest"
+
+@pytest.fixture(scope="class")
+def triton_server(model_repository, request):
+    port = 0
+    try:
+        sock = socket.socket()
+        sock.bind(("", 0))
+        port = sock.getsockname()[1]
+        sock.close()
+    except socket.error as e:
+        if e.errno != errno.EADDRINUSE:
+            raise Exception(f"Not expected exception found in port manager: {e.errno}")
+    image_name = request.config.getoption("--image")
+    image_name = image_name if image_name is not None else "tritonserver:latest"
+    gpu = '--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* )' if request.config.getoption("--gpu") else ""
+    subprocess.run(
+        f'docker run -p {port}:8001 -d -v {model_repository}:/model_repository --name={CONTAINER_NAME} {gpu}  {image_name} bin/tritonserver --model-repository /model_repository',
+        capture_output=True,
+        shell=True,
+    )
+    subprocess.run(["sleep", "3"])
+    subprocess.run(["docker", "logs", CONTAINER_NAME])
```

Comment:
add printing the container logs for debugging purposes, currently there is not clue why container is not started